src/routes/MetaDetails/styles.less

```diff
@@ -17,6 +17,34 @@
     flex-direction: column;
     width: 100%;
     height: 100%;
+    padding-left: var(--safe-area-inset-left, 0px);
+    padding-right: var(--safe-area-inset-right, 0px);
+    box-sizing: border-box;
+
+    --navbar-assumed-height: 84px;
```

Comment:
Here the variables are indeed all static, but I think that for consistency with the other files it may make sense to keep them css vars.
Let me know if you agree or want me to change it to preprocessor variables anyway.